<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>照片查看器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/full.png</normaloff>:/images/full.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 226, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tBtn_large">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>放大</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/large.png</normaloff>:/images/large.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtn_left">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>上一个(向左键)</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
          </property>
          <property name="shortcut">
           <string>Left</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtn_full">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>幻灯片放映</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/full.png</normaloff>:/images/full.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtn_right">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>下一个(向右键)</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
          </property>
          <property name="shortcut">
           <string>Right</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtn_toLeft">
          <property name="toolTip">
           <string>逆时针旋转</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/toLeft.png</normaloff>:/images/toLeft.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtn_toRight">
          <property name="toolTip">
           <string>顺时针旋转</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/toRight.png</normaloff>:/images/toRight.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtn_del">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>删除</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/del.png</normaloff>:/images/del.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_0">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="act_open"/>
    <addaction name="act_saveAs"/>
    <addaction name="act_close"/>
    <addaction name="separator"/>
    <addaction name="act_clear"/>
    <addaction name="separator"/>
    <addaction name="act_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="act_about"/>
    <addaction name="act_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="act_large"/>
    <addaction name="act_small"/>
    <addaction name="act_toLeft"/>
    <addaction name="act_toRight"/>
   </widget>
   <addaction name="menu_0"/>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_open"/>
   <addaction name="act_saveAs"/>
   <addaction name="act_large"/>
   <addaction name="act_small"/>
   <addaction name="act_toLeft"/>
   <addaction name="act_toRight"/>
   <addaction name="act_about"/>
  </widget>
  <action name="act_open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open_new.png</normaloff>:/images/open_new.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开(O)</string>
   </property>
  </action>
  <action name="act_saveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;S)</string>
   </property>
  </action>
  <action name="act_close">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;W)</string>
   </property>
  </action>
  <action name="act_quit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="act_about">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="act_Qt">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Qt.png</normaloff>:/images/Qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="act_large">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/large.png</normaloff>:/images/large.png</iconset>
   </property>
   <property name="text">
    <string>放大(&amp;L)</string>
   </property>
  </action>
  <action name="act_small">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/small.png</normaloff>:/images/small.png</iconset>
   </property>
   <property name="text">
    <string>缩小(&amp;S)</string>
   </property>
  </action>
  <action name="act_toLeft">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/toLeft.png</normaloff>:/images/toLeft.png</iconset>
   </property>
   <property name="text">
    <string>向左旋转90°(L)</string>
   </property>
  </action>
  <action name="act_toRight">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/toRight.png</normaloff>:/images/toRight.png</iconset>
   </property>
   <property name="text">
    <string>向右旋转90°(R)</string>
   </property>
  </action>
  <action name="act_clear">
   <property name="text">
    <string>清除最近打开列表</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
